schemas/context/viewability-details.schema.json

```diff
@@ -0,0 +1,102 @@
+{
+  "meta:license": [
+    "Copyright 2018 Adobe Systems Incorporated. All rights reserved.",
+    "This work is licensed under a Creative Commons Attribution 4.0 International (CC BY 4.0) license",
+    "you may not use this file except in compliance with the License. You may obtain a copy",
+    "of the License at https://creativecommons.org/licenses/by/4.0/"
+  ],
+  "$id": "https://ns.adobe.com/xdm/context/viewability-details",
+  "$schema": "http://json-schema.org/draft-06/schema#",
+  "title": "Viewability",
+  "type": "object",
+  "description":
+  "Description of viewability measured properties of an advertisement",
+  "definitions": {
+    "viewabilityDetails": {
+      "properties": {
+        "xdm:windowDepth": {
+          "title": "Window Depth",
+          "type": "integer",
+          "description":
+          "The number of iFrames the ad is displayed inside of.",
+          "minimum": 0
+        },
+        "xdm:viewportHeight": {
+          "title": "Viewport Height",
+          "type": "integer",
+          "description":
+          "The number of vertical pixels of the window the ad is displayed in as measured by the viewability library.",
```

Comment:
"the viewability library." is specific to the Adobe implementation, can we express this in a more general way.